src/data/PredictorD2ApiRepository.ts

```diff
@@ -41,7 +41,7 @@ export class PredictorD2ApiRepository implements PredictorRepository {
             this.api.models.predictors.get({
                 filter: { id: { in: ids } },
                 paging: false,
-                fields: predictorFields,
+                fields: { ...predictorFields, $owner: true },
```

Comment:
If im not mistaken, `$owner: true` is added for the `organisationUnitDescendants` field.
Why not add it to the `predictorFields` const?
Same for line 77.